### PR TITLE
First autograding testcase now is pre-opened

### LINKED ISSUE #2316: student view single autograding test case should be pre-opened
For example, "Grades Released Homework"
Should not have to click on "Test 1 Lab 1 Checkpoint 1" to see the student/expected output

(The hidden # of submissions check is probable confusing the logic again)

### PR DESCRIPTION
**Pull request by saxocellphone**
I'm not really sure why this was put in place to begin with, but there is a specific if-statement that says don't open the testcase if it's the only one.

Closes #2316

### CODE REVIEW

**Comment by MasterOdin** [comment]
Does this change mean that the first testcase will always be open?

**Comment by CouleeApps** [comment]
Oh yep. Didn't notice that because all of the sample data gradeables have either 1 testcase or the first testcase hiding output.

**Comment by KevinMackenzie** [comment]
This looks good to me.  Are there any unresolved issues with this?  From the issue it looks like opening the first test case should only happen if there is one visible test case.

**Comment by saxocellphone** [comment]
Should be ready to go now

File: site/app/templates/autograding/AutoResults.twig [hunk 1]
```diff
@@ -69,14 +69,22 @@
                         {% endif %}
                     </div>
                 {% endif %}
-            {% elseif has_badges %}
-                <div class="no-badge"></div>
-            {% endif %}
-            {# /Badge #}
-
-            <h4>
```
Comment (CouleeApps) [inline site/app/templates/autograding/AutoResults.twig]: This <h4> is removed without being replaced... so now there's just a </h4> floating in the wild

File: site/app/templates/autograding/AutoChecks.twig [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
-        <iframe src="{{ core.buildUrl({"component": "misc", "page": "display_file", "dir": "results", "file": check.name, "path": check.path}) }}" width="95%" height="1200px" style="border: 0"></iframe>
-    {% else %}
-        <br>
-        <div style="height:23px; float:right;">
-            <a id="show_char_{{ index }}_{{ loop.index0 }}" class="btn btn-default" style="float:right;" onclick="changeDiffView('container_{{ index }}_{{ loop.index0 }}', '{{ gradeable.getId() }}', '{{ who }}', '{{ index }}', '{{ loop.index0 }}', 'white_space_helper_{{ index }}')">Visualize whitespace characters</a>
-        </div>
-        <div><span style="margin-left: 20px" class="line_code" id='white_space_helper_{{ index }}'></span></div>
-        <div class="box-block">
-
+    {% if check.actual %}
```
Comment (CouleeApps) [inline site/app/templates/autograding/AutoChecks.twig]: `{% for check in checks if check.actual %}` is cleaner

**Comment by bmcutler** [comment]
@CouleeApps do you approve?

**Comment by CouleeApps** [comment]
That is one heck of a revert... Let me make sure it works and then i'll approve it

**Pull request merged**